src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -55,7 +56,7 @@ using namespace std::chrono;
 
 std::atomic<int> highPriorityTaskCount;
 static int connFd;
-static volatile int currentFESession;
+static std::atomic<int> currentFESession;
```

Comment:
What is the reason for this change from `volatile` to `std::atomic`?

Reply:
We've created separate threads for the UI frontend and the existing functionality, therefore its shared variable, To ensure thread safety, I switched it to std::atomic. otherwise getting issue